src/pages/Products/components/ProductsError.jsx

```diff
@@ -0,0 +1,27 @@
+import React, { useEffect } from 'react'
+import { Typography } from '@mui/material'
+
+import { useProductsDataContext } from '../../../context/ProductsDataContext'
+import Localizator from '../../../common/components/Localizator'
+import { useGetProducts } from '../../../services/getProducts/getProducts'
+
+const ProductsError = () => {
+  const { prodsErr } = useGetProducts()
+  const { error, setProductsDataError } = useProductsDataContext()
+
+  useEffect(() => {
+    if (prodsErr) {
+      setProductsDataError()
+    }
+  }, [prodsErr])
+
+  if (error) {
+    return (
```

Comment:
I think it can be moved to shared components in this way:
1. Create NotificationContext 
2. Add reducer and action to set notification state
3. Create a provider which renders notifications using [Snackbar component](https://mui.com/material-ui/react-snackbar/#main-content)
4. Wrap your app with a notification provider